aws-master-account-iam-groups.yaml

```diff
@@ -70,68 +70,67 @@ Resources:
+  AssumeAWSAdminRolePolicy:
     Type: AWS::IAM::ManagedPolicy
     Properties:
-      ManagedPolicyName: AssumeDeveloperRolePolicy
+      ManagedPolicyName: AssumeAWSAdminRolePolicy
       PolicyDocument:
         Version: '2012-10-17'
         Statement:
         - Effect: Allow
           Action:  [
                 "sts:AssumeRole"
             ]
-          Resource: !Join [ "", [ 'arn:aws:iam::', '*', ':role/DeveloperRole'] ]
+          Resource: !Join [ "", [ 'arn:aws:iam::', '*', ':role/AWSAdminRole'] ]
           Condition:
             BoolIfExists:
                 aws:MultiFactorAuthPresent: 'true'
 
-
-  AssumeReadOnlyRolePolicy:
+  AssumeDeveloperRolePolicy:
     Type: AWS::IAM::ManagedPolicy
     Properties:
-      ManagedPolicyName: AssumeReadOnlyRolePolicy
+      ManagedPolicyName: AssumeDeveloperRolePolicy
       PolicyDocument:
         Version: '2012-10-17'
         Statement:
         - Effect: Allow
           Action:  [
                 "sts:AssumeRole"
             ]
-          Resource: !Join [ "", [ 'arn:aws:iam::', '*', ':role/ReadOnlyRole'] ]
+          Resource: !Join [ "", [ 'arn:aws:iam::', '*', ':role/DeveloperRole'] ]
           Condition:
             BoolIfExists:
                 aws:MultiFactorAuthPresent: 'true'
-  
-  AssumeNetworkAdminRolePolicy:
+
+
+  AssumeReadOnlyRolePolicy:
     Type: AWS::IAM::ManagedPolicy
     Properties:
-      ManagedPolicyName: AssumeNetworkAdminRolePolicy
+      ManagedPolicyName: AssumeReadOnlyRolePolicy
       PolicyDocument:
         Version: '2012-10-17'
         Statement:
         - Effect: Allow
           Action:  [
                 "sts:AssumeRole"
             ]
-          Resource: !Join [ "", [ 'arn:aws:iam::', '*', ':role/NetworkAdminRole'] ]
+          Resource: !Join [ "", [ 'arn:aws:iam::', '*', ':role/ReadOnlyRole'] ]
           Condition:
             BoolIfExists:
                 aws:MultiFactorAuthPresent: 'true'
   
-  AssumeAWSAdminRolePolicy:
-    Type: AWS::IAM::ManagedPolicy
-    Properties:
-      ManagedPolicyName: AssumeAWSAdminRolePolicy
-      PolicyDocument:
-        Version: '2012-10-17'
-        Statement:
-        - Effect: Allow
-          Action:  [
-                "sts:AssumeRole"
-            ]
-          Resource: !Join [ "", [ 'arn:aws:iam::', '*', ':role/AWSAdminRole'] ]
-          Condition:
-            BoolIfExists:
-                aws:MultiFactorAuthPresent: 'true'
-  
\ No newline at end of file
+  # AssumeNetworkAdminRolePolicy:
+  #   Type: AWS::IAM::ManagedPolicy
+  #   Properties:
+  #     ManagedPolicyName: AssumeNetworkAdminRolePolicy
+  #     PolicyDocument:
+  #       Version: '2012-10-17'
+  #       Statement:
+  #       - Effect: Allow
+  #         Action:  [
+  #               "sts:AssumeRole"
+  #           ]
+  #         Resource: !Join [ "", [ 'arn:aws:iam::', '*', ':role/NetworkAdminRole'] ]
+  #         Condition:
+  #           BoolIfExists:
+  #               aws:MultiFactorAuthPresent: 'true'
```

Comment:
@maybealexandra can you please delete this in a separate commit instead of commenting it out?